libs/components/forms/src/lib/modules/input-box/input-box.component.scss

```diff
@@ -394,9 +420,32 @@ sky-input-box {
       }
 
       textarea.sky-form-control {
-        margin-top: 0;
+        margin-top: -23px;
```

Comment:
The -23px and 26 px is what the standard input uses except for it uses padding because we do not have to worry about the label overlap issue.